src/app/components/ThemeProvider/withThemeProvider.tsx

```diff
@@ -0,0 +1,34 @@
+import { ThemeProvider as EmotionThemeProvider } from '@emotion/react';
+import mergeDeepLeft from 'ramda/src/mergeDeepLeft';
+
+import * as colours from './colours';
+import * as mq from './mediaQueries';
+import * as spacings from './spacings';
+
+import { BrandColours } from '../../interfaces';
+
+interface ServiceTheme {
+  colours: BrandColours;
+}
+
+type Props = {
+  children: JSX.Element | JSX.Element[];
+};
+
+const withThemeProvider = (serviceTheme: ServiceTheme) => {
+  const ThemeProvider: React.FC<Props> = ({ children }: Props) => (
```

Comment:
If we're using `React.FC`, we shouldn't need to provide it with `Props` if its just the children we're typing. Before React 18, `React.FC` includes type definitions for children by default. This changes in React 18, so we'd need to use something like `PropsWithChildren`, or typing the children as `children: React.ReactNode`.